config/locales/en.yml

```diff
@@ -55,6 +55,10 @@ en:
               too_short: "password is too short"
             address:
               blank: "address is blank"
+            password_confirmation:
+              confirmation: "Wrong passwordaa cornfirmation"
```

Comment:
Are you okey? Hello. It should be "Wrong password confirmation"

Reply:
already fixed